Tests/UnitTests/Core/Vertexing/AdaptiveMultiVertexFitterTests.cpp

```diff
@@ -54,6 +55,9 @@ namespace Test {
 using namespace Acts::UnitLiterals;
 using Acts::VectorHelpers::makeVector4;
 
+// Set up logger
+ACTS_LOCAL_LOGGER(Acts::getDefaultLogger("AMVFitterTests", Acts::Logging::INFO))
```

Comment:
This is not actually needed anymore I think. Just do:

```suggestion
auto logger = Acts::getDefaultLogger("AMVFitterTests", Acts::Logging::INFO)
```

Reply:
Scratch that, never mind.